.github/workflows/lwjgl.yml

```diff
@@ -60,11 +60,11 @@ jobs:
     name: Linux Cross
     runs-on: ubuntu-latest
     container:
-      image: ubuntu:18.04
+      image: ubuntu:22.04
```

Comment:
Why? Because 18.04 does not have `gcc-powerpc64le-linux-gnu libc6-dev-ppc64el-cross` packages